src/main.js

```diff
@@ -105,6 +107,11 @@ const store = new Vuex.Store({
 });
 
 Vue.mixin({
+  methods: {
+    fluent(...args) {
```

Comment:
Out of curiosity, what's the argument behind putting this in the global methods mixin, rather than in the /components/_mixins/ directory?

Reply:
Eventually we'll be using this in every file with user-facing strings (which will be close to all of them)

Reply:
Ah that makes sense. Cool.